projects/ngneat/reactive-forms/src/lib/persistManager.ts

```diff
@@ -0,0 +1,6 @@
+import { Observable } from 'rxjs';
+
+export interface PersistManager<T> {
+  setValue(key: string, data: T): void;
```

Comment:
`setValue` could also be async

Reply:
That's true, but the concrete manager that implements `setValue` could manage the async requests internally. I am not sure what exactly the library should do with the response from the requests, besides ensuring they are run in order etc.

Reply:
We need to handle race conditions.

Reply:
I adapted the interface, please let me know if it is ok.